api/src/main/java/org/openmrs/notification/impl/MessageServiceImpl.java

```diff
@@ -258,7 +258,7 @@ public Message prepareMessage(Template template) throws MessageException {
 	@Transactional(readOnly = true)
 	public Message prepareMessage(String templateName, Map data) throws MessageException {
 		try {
-			Template template = (Template) getTemplatesByName(templateName).get(0);
+			Template template = getTemplatesByName(templateName).get(0);
```

Comment:
Was curios to see how you ended up with this line,good i saw it in the above class.